easybuild/easyconfigs/m/Minimac2/Minimac2-2014.9.15-intel-2015b.eb

```diff
@@ -0,0 +1,29 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "Tarball"
```

Comment:
@rtripath89 wouldn't it be better to recompile rather using the shipped binaries? similar to `Minimac`, see https://github.com/hpcugent/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/m/Minimac/Minimac-20140110-goolf-1.4.10.eb ?